service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -75,14 +93,14 @@ void submittedEventShouldNotifyHmctsAdminWhenCtscToggleIsDisabled() throws Excep
         verify(notificationClient).sendEmail(
             C2_UPLOAD_NOTIFICATION_TEMPLATE,
             "admin@family-court.com",
-            expectedNotificationParams(),
+            expectedNotificationParams().toMap(mapper),
```

Comment:
Recently we've been omitting 'expected template data' at integration test level and using `anyMap()`. As long as it's fully tested in unit tests we only really care at this point wether the correct template was sent and to the right person. 

No need to action this now since you have it all done.

Reply:
cool. thanks will remember this for next time. :)